### DISCUSSION TITLE
Bundling pgzhelper with Mu?

### DISCUSSION THREAD

**Discussion by eugene-bce** [mu-editor/mu#1800] [Ideas & New Features]
https://www.aposteriori.com.sg/pygame-zero-helper/

Pgzhelper adds some significant enhancement to Pygame Zero. It would be fantastic if it came bundled with Mu, particularly as pgzhelper isn't available via the package third party packages.

**Comment by roboticsware** [comment]
I integrated the module to pygame zero mode. Check out the version at https://github.com/roboticsware/mu . Thanks. @eugene-bce
